src/inc/msquic_posix.h

```diff
@@ -38,6 +38,8 @@
 #include <errno.h>
 #include "quic_sal_stub.h"
 
+#define QUIC_INLINE static inline
```

Comment:
Could it backfire to have `QUIC_INLINE` always be static under the hood?
It there situations where we need only inline in a `forceinline` meaning, and `QUIC_INLINE` would be confusing?

(since it is more meant as a "inline in header")?

Reply:
To partially answer your question, there is a separate `FORCEINLINE` macro defined that I also had to fix.

Reply:
In that case, maybe `QUIC_HEADER_INLINE`?
Or actually, is there a problem with `static inline` for all platforms?

In any case, the PR is an improvement over the current state.

Reply:
I don't see a substantial win with QUIC_HEADER_INLINE, and now the code is straightforward to refactor in the future if a significantly better name occurs to us.